ml-agents-envs/mlagents_envs/gym_wrapper.py

```diff
@@ -0,0 +1,166 @@
+from mlagents_envs.base_env import (
+    BaseEnv,
+    DecisionSteps,
+    TerminalSteps,
+    BehaviorSpec,
+    BehaviorName,
+    AgentId,
+    ActionType,
+)
+from mlagents_envs.exception import UnityActionException, UnityObservationException
+
+from typing import List, Tuple
+
+import numpy as np
+
+try:
+    import gym
+except ImportError:
+    raise ImportError("gym is not installed, gym required to use the GymToUnityWrapper")
```

Comment:
A safer way to do this would be to raise the error at runtime instead of import time:

```
try:
    import gym
    _GYM_IMPORTED = True
except ImportError:
    _GYM_IMPORTED = False

...

class GymToUnityWrapper(BaseEnv):
    def __init__(self, gym_env, name=None):
        if not _GYM_IMPORTED:
            raise RuntimeError("gym is not installed, gym required to use the GymToUnityWrapper")
```